config/manifests/bases/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -59,4 +59,4 @@ spec:
     name: Red Hat
     url: https://github.com/identitatem
   replaces: PREV_CATALOG_VERSION
-  version: 0.2.0
+  version: 0.0.1
```

Comment:
Any reason why we bumped the version back down?

Reply:
Going to leave it as 0.0.1 in the source permanently. The real version gets passed in via VERSION env var in the GitHub actions.

Reply:
Does this mean that when CSVs are generated and pushed to the repo that they will always be versioned with 0.0.1? Or that there is at least a possibility that it will be pushed with the wrong version? If so, I would suggest to _not_ do this because of the possibility to mess up the metadata/upgrade graph.

Reply:
If this version number is never to be changed, please add a comment about this and that the only version number that should be changed is in the github action and point to those in repo.  It will help code maintenance in the future

Reply:
Yes, the bundle manifest in the repo should always be version 0.0.1 and the image should always be `quay.io/identitatem/idp-mgmt-operator:latest`. The latter is more important because the downstream automation is looking for the "latest" specifically. The version it really doesn't matter if somebody checks something else in because we always overwrite it when we're building and publishing our official images. That said, agree, we should add a comment to not change it.
@chrisahl the version number in the GitHub action shouldn't change either. My thought was anything coming out of push to main/stable can just always be 0.0.1, and we get our official "versions" from tagging. Please let me know what breaks with this approach. It just feels like unnecessary overhead to me to have versions hardcoded at all when we aspire to release frequently, but if something's broken I of course need to understand that. :)
In my head at least, these changes bring our upstream and downstream processes closer together, where `make bundle` is NOT run at the time we're building and pushing our images in either case, but rather we are just patching in the correct version info over whatever is in the repo.

Reply:
Per our conversation in our meeting on 2/2, I believe we are all aligned on this. Please shout if not.